spec/spec_helper.rb

```diff
@@ -0,0 +1,9 @@
+require 'chefspec'
+require 'chefspec/berkshelf'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.